guides/source/active_support_instrumentation.md

```diff
@@ -517,6 +517,16 @@ INFO. Cache stores may add their own keys
 }
 ```
 
+#### rotation.active_support
```

Comment:
@kaspth do we still need an entry here, even if there is nothing in the payload?